runtime/common/voter-bags/src/main.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+
+//! Make the set of voting bag thresholds to be used in `voter_bags.rs`.
```

Comment:
Can we add a little doc to say how often this should be run and stuff?

Reply:
https://github.com/paritytech/polkadot/pull/3413/commits/387727b7eb63412ca2d995f402c3a53ce9104176